partner_compassion/views/res_partner_view.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<record id="act_partner_survey_input" model="ir.actions.act_window">
+    <field name="res_model">survey.user_input</field>
+    <field name="view_type">form</field>
+    <field name="name">Surveys</field>
+    <field name="view_mode">tree,form</field>
+    <field name="context">{'search_default_partner_id': active_id, 'default_partner_id': active_id}</field>
+</record>
+
+<record model="ir.ui.view" id="view_partner_form">
+    <field name="name">Partner Form with survey answers</field>
+    <field name="model">res.partner</field>
+    <field name="inherit_id" ref="base.view_partner_form"/>
+    <field name="arch" type="xml">
+        <div class="oe_button_box" position="inside">
```

Comment:
This view inheritance should be merged in the main view defined in partner_compassion_view. No need to have so many different views.

Reply:
I don't understand which main view you are talking about in partner_compassion_view